### PR TITLE
Issue 289 various endpoints

### LINKED ISSUE #289: Various endpoints
**API Token scopes**
- [x] List Scopes endpoint: https://canvas.instructure.com/doc/api/api_token_scopes.html#method.scopes_api.index

**Account Notifications**
- [x] Show a global notification: https://canvas.instructure.com/doc/api/account_notifications.html#method.account_notifications.show

- [x] Update a global notification: https://canvas.instructure.com/doc/api/account_notifications.html#method.account_notifications.update

**Account Reports**
- [x] Start A report: https://canvas.instructure.com/doc/api/account_reports.html#method.account_reports.create

- [x]  Status of a Report: https://canvas.instructure.com/doc/api/account_reports.html#method.account_reports.show

- [x] Delete a Report: https://canvas.instructure.com/doc/api/account_reports.html#method.account_reports.destroy

### PR DESCRIPTION
**Pull request by jrsilveti**
# Proposed Changes
Added coverage for: 
List Scopes endpoint
Show a global notification
Update a global notification
Start A report
Status of a Report
Delete a Report

Resolves #289

### CODE REVIEW

File: canvasapi/account.py [hunk 1]
```diff
@@ -1702,14 +1804,66 @@ def update_role(self, role, **kwargs):
 @python_2_unicode_compatible
 class AccountNotification(CanvasObject):
     def __str__(self):  # pragma: no cover
```
Comment (Thetwam) [inline canvasapi/account.py]: remove `# pragma: no cover`, and add test (if applicable)

File: canvasapi/account.py [hunk 2]
```diff
@@ -1702,14 +1804,66 @@ def update_role(self, role, **kwargs):
...
+        <https://canvas.instructure.com/doc/api/account_notifications.html#method.account_notifications.update>`_
+
+        :param account_notification: The notification to update with.
+        :type account_notification: dict
+
+        :rtype: :class:`canvasapi.account.AccountNotification`
+        """
+        required_key_list = ["subject", "message", "start_at", "end_at"]
+        required_keys_present = all(
+            (x in account_notification for x in required_key_list)
```
Comment (Thetwam) [inline canvasapi/account.py]: Please remove the extra pair of parentheses.

File: canvasapi/account.py [hunk 3]
```diff
@@ -1702,14 +1804,66 @@ def update_role(self, role, **kwargs):
...
+
+        :calls: `PUT /api/v1/accounts/:account_id/account_notifications/:id \
+        <https://canvas.instructure.com/doc/api/account_notifications.html#method.account_notifications.update>`_
+
+        :param account_notification: The notification to update with.
+        :type account_notification: dict
+
+        :rtype: :class:`canvasapi.account.AccountNotification`
+        """
+        required_key_list = ["subject", "message", "start_at", "end_at"]
```
Comment (Thetwam) [inline canvasapi/account.py]: The [Canvas Docs](https://canvas.instructure.com/doc/api/account_notifications.html#method.account_notifications.update) don't list these as required parameters. Did you find that they were actually required in testing?

File: canvasapi/account.py [hunk 4]
```diff
@@ -1702,14 +1804,66 @@ def update_role(self, role, **kwargs):
...
+            "accounts/{}/account_notifications/{}".format(self.account_id, self.id),
+            _kwargs=combine_kwargs(**kwargs),
+        )
+
+        return AccountNotification(self._requester, response.json())
 
 
 @python_2_unicode_compatible
 class AccountReport(CanvasObject):
     def __str__(self):  # pragma: no cover
```
Comment (Thetwam) [inline canvasapi/account.py]: Remove `# pragma: no cover` and add test

File: tests/fixtures/account.json [hunk 5]
```diff
@@ -459,6 +473,96 @@
...
+    		"attachment": {
+    		    "id": 1432,
+    		    "uuid": "CCVh71fQEPsDqc4VOoGBr5llCbzAeLEpBcWcAe88",
+    		    "folder_id": 53,
+    		    "display_name": "zero_activity_csv_13_Aug_2019_420190813-16-ecpli.csv",
+    		    "filename": "zero_activity_csv_13_Aug_2019_120190813-16-8jbpun.csv",
+    		    "workflow_state": "processed",
+    		    "upload_status": "success",
+    		    "content-type": "text/csv",
+    		    "url": "https://canvas.dev.cdl.ucf.edu/files/1432/download?download_frd=1&verifier=CCVh71fQEPsDqc4VOoGBr5llCbzAeLEpBcWcAe88",
```
Comment (Thetwam) [inline tests/fixtures/account.json]: Don't use real data (even from our fake dev server) for this. Anonymize these blocks.

File: canvasapi/account.py (hunk 3 above)
Comment (jrsilveti) [reply canvasapi/account.py]: Yes, when manually creating a notification in canvas those fields were required.

File: canvasapi/account.py (hunk 1 above)
Comment (jrsilveti) [reply canvasapi/account.py]: Added test.

File: canvasapi/account.py (hunk 2 above)
Comment (jrsilveti) [reply canvasapi/account.py]: Removed.

File: canvasapi/account.py (hunk 4 above)
Comment (jrsilveti) [reply canvasapi/account.py]: Added test.

File: tests/fixtures/account.json (hunk 5 above)
Comment (jrsilveti) [reply tests/fixtures/account.json]: Anonymized data.

**Pull request merged**